packages/strapi-admin/admin/src/containers/LeftMenu/useMenuSections.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react';
+import { useUser } from 'strapi-helper-plugin';
+import { useSelector, useDispatch } from 'react-redux';
+import getCtOrStLinks from './utils/getCtOrStLinks';
+import getPluginSectionLinks from './utils/getPluginSectionLinks';
+import getGeneralLinks from './utils/getGeneralLinks';
+import { setCtOrStLinks, setSectionLinks, toggleIsLoading } from './actions';
+import useSettingsMenu from '../../hooks/useSettingsMenu';
+import toPluginLinks from './utils/toPluginLinks';
+import selectMenuLinks from './selectors';
+
+const useMenuSections = (plugins, shouldUpdateStrapi) => {
```

Comment:
ℹ️ this is the important hook that gather and gives the links to the leftMenu. It holds the logic (permissions + formatting) of the different links